### ISSUE TITLE
Filter ADE to retain specific images belonging to specific class

### ISSUE THREAD

**Issue by sparshgarg23** [CSAILVision/ADE20K#41]
Hi,is it possible to filter the ADE20K dataset in such a way that only images that have a specific class are included
For example in the dataset there are 318 images corresponding to the class bridge.I would like to retain only these images and ignore the rest of them.Is that possible,and if yes how would you suggest i go about it.

**Comment by macTracyHuang** [comment]
> Hi,is it possible to filter the ADE20K dataset in such a way that only images that have a specific class are included For example in the dataset there are 318 images corresponding to the class bridge.I would like to retain only these images and ignore the rest of them.Is that possible,and if yes how would you suggest i go about it.

Yes, you can refer to my [repo](https://github.com/macTracyHuang/ADE20K_To_COCO)
